src/Backoffice/Employee/App/Controllers/ListEmployeesController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Illuminate\Routing\Controller;
+use Lightit\Backoffice\Employee\App\Resources\EmployeeResource;
+use Lightit\Backoffice\Employee\Domain\Actions\ListEmployeesAction;
+
+class ListEmployeesController extends Controller
```

Comment:
```suggestion
class ListEmployeesController
```
you arent using anything from Controller class so no need to extend it